example_configs/presto.yml

```diff
@@ -0,0 +1,48 @@
+lowercaseOutputName: true
+lowercaseOutputLabelNames: true
+whitelistObjectNames: ["com.facebook.presto.hive:type=NamenodeStats,*", "com.facebook.presto.hive.metastore:type=ThriftHiveMetastore,*", "com.facebook.presto.failureDetector:name=HeartbeatFailureDetector,*", "com.facebook.presto.execution:name=QueryManag\
+er,*", "com.facebook.presto.execution:name=TaskManager,*", "com.facebook.presto.memory:type=ClusterMemoryPool,*", "com.facebook.presto.memory:name=ClusterMemoryManager,*"]
+blacklistObjectNames: []
+rules:
+  - pattern: "com.facebook.presto.hive<type=NamenodeStats, name=hive><>(.+): (.*)"
+    name: "presto:hive_namenodestats_$1"
```

Comment:
metric names from instrumentation should not contain colons

Reply:
Thanks, will update.